src/components/tier.jsx

```diff
@@ -63,12 +48,21 @@ class Tier extends React.Component {
 
 
   render() {
+
+    const items = this.state.items;
+
+    let itemsArray = [];
+
+    for (let key in items) {
+      let itemId = items[key];
+      itemsArray.push(<Item key={itemId} itemId={itemId}/>)
+    }
+
     return (
       <div>
-        <div className="item">
+        <div className="tier ${this.state.tierId">
           Tier {this.state.tierId}
-          <Item itemId={this.state.items.item1}/>
-          <Item itemId={this.state.items.item2}/>
+          {itemsArray}
         </div>
       </div>
     )
```

Comment:
✓ Your module is pretty complex and I really like how you handled the data flow. Great work so far!!